modules/core/src/v2/wallet.ts

```diff
@@ -1768,20 +1768,20 @@ export class Wallet {
    * @return {*}
    */
   async signTransaction(params: WalletSignTransactionOptions = {}): Promise<SignedTransaction> {
-    const txPrebuild = params.txPrebuild;
+    const { txPrebuild } = params;
     if (!txPrebuild || typeof txPrebuild !== 'object') {
       throw new Error('txPrebuild must be an object');
     }
     const presign = await this.baseCoin.presignTransaction(params);
```

Comment:
@sangaman actually, due to a "lucky accident", `presign === params` here and the `params.pubs` assignment later on also modifies `presign`

still we should prefer the new variant :)